main.tf

```diff
@@ -347,7 +347,7 @@ resource "aws_security_group_rule" "provisioner_secgrp_ingress_443" {
 # Provisioner
 #------------------------------------------------------------------------------
 module "ranchhand" {
-  source = "github.com/dominodatalab/ranchhand?ref=v0.1.3-rc1"
+  source = "github.com/dominodatalab/ranchhand?ref=v0.1.4-rc1"
```

Comment:
@steved The release is yet to be created. Waiting for https://github.com/dominodatalab/ranchhand/pull/32 to be merged.